pygsti/algorithms/fiducialpairreduction.py

```diff
@@ -290,11 +298,12 @@ def _get_number_amplified(m0, m1, len0, len1, verb):
                       for iEStr in range(nEStrs)]
     return listOfAllPairs
 
-
+#@_deprecated_fn('find_sufficient_fiducial_pairs_per_germ_power')
```

Comment:
I thought we did end up deprecating per germ power? Or are we keeping it for the per-germ power random "robustness" feature

Reply:
Good question, I'm not sure if the right move is deprecation or not here. The justification for keeping it around would be the robustness conjecture, but even that claim hasn't actually been tested yet.

Reply:
We can keep it for now, and consider testing robustness vs deprecation later.